server/platform/services/cache/provider.go

```diff
@@ -91,8 +92,9 @@ func NewRedisProvider(opts *RedisOptions) (Provider, error) {
 		Password:          opts.RedisPassword,
 		SelectDB:          opts.RedisDB,
 		ForceSingleClient: true,
-		CacheSizeEachConn: 16 * (1 << 20), // 16MiB local cache size
-		//TODO: look into MaxFlushDelay
+		CacheSizeEachConn: 32 * (1 << 20), // 32MiB local cache size
+		DisableCache:      opts.DisableCache,
+		MaxFlushDelay:     250 * time.Microsecond,
```

Comment:
We could leave a comment to explain this. Ideally, it would be nice to include some benchmarks/results somewhere.